File: output/ifc4x3_dev_73740fe4/entity/IfcResourceTime.java
package com.topoplanet.ifc4x3_dev_73740fe4.entity;
import com.topoplanet.test.type.*;


public class IfcResourceTime extends IfcSchedulingTime {

    public IfcDuration ScheduleWork; // OPTIONAL
    public IfcPositiveRatioMeasure ScheduleUsage; // OPTIONAL
    public IfcDateTime ScheduleStart; // OPTIONAL
    public IfcDateTime ScheduleFinish; // OPTIONAL
    public IfcLabel ScheduleContour; // OPTIONAL
    public IfcDuration LevelingDelay; // OPTIONAL
    public IfcBoolean IsOverAllocated; // OPTIONAL
    public IfcDateTime StatusTime; // OPTIONAL
    public IfcDuration ActualWork; // OPTIONAL
    public IfcPositiveRatioMeasure ActualUsage; // OPTIONAL
    public IfcDateTime ActualStart; // OPTIONAL
    public IfcDateTime ActualFinish; // OPTIONAL
    public IfcDuration RemainingWork; // OPTIONAL
    public IfcPositiveRatioMeasure RemainingUsage; // OPTIONAL
    public IfcPositiveRatioMeasure Completion; // OPTIONAL
}

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcSchedulingTime/IfcResourceTime/IfcResourceTime.vb
Public Class IfcResourceTime Inherits IfcSchedulingTime
    Public ScheduleWork As IfcDuration
    Public ScheduleUsage As IfcPositiveRatioMeasure
    Public ScheduleStart As IfcDateTime
    Public ScheduleFinish As IfcDateTime
    Public ScheduleContour As IfcLabel
    Public LevelingDelay As IfcDuration
    Public IsOverAllocated As IfcBoolean
    Public StatusTime As IfcDateTime
    Public ActualWork As IfcDuration
    Public ActualUsage As IfcPositiveRatioMeasure
    Public ActualStart As IfcDateTime
    Public ActualFinish As IfcDateTime
    Public RemainingWork As IfcDuration
    Public RemainingUsage As IfcPositiveRatioMeasure
    Public Completion As IfcPositiveRatioMeasure
End Class
